tests/acceptance/docker/Makefile

```diff
@@ -257,11 +257,3 @@ clean-files:
 
 .PHONY: clean
 clean: clean-docker-container clean-docker-volumes clean-dev-docker-image clean-files ## clean all
-
-../../../vendor-bin/behat/composer.lock: ../../../vendor-bin/behat/composer.json
-	@echo behat composer.lock is not up to date.
-	@composer update --no-progress -d ../../../vendor-bin/behat
-
-../../../composer.lock: ../../../composer.json
-	@echo composer.lock is not up to date.
-	@composer update --no-progress -d ../../../
```

Comment:
Good findings :+1: . Indeed it will use the host's php to do the composer update before running tests creating composer.lock using the host php and making it incompatible with container php (if there are different php in host and container).

But AFAIK, this was added to update the lock file if there is already the old lock file which can cause the test run script to fail. First of all, adding those commands in `tests/acceptance/docker/Makefile` was not the right choice and second is that the command should rather be `composer update --no-progress --lock` to only update the lock file not the packages.
So, could you try to run tests with old lock file (with older package versions) and check if the test can run in the container? I guess, it will throw an error.

One solution would be to add `@composer update --no-progress --lock` in the root Makefile
```diff
vendor-bin/behat/composer.lock: vendor-bin/behat/composer.json
	@echo behat composer.lock is not up to date.
+   @composer update --no-progress --lock

composer.lock: composer.json
	@echo composer.lock is not up to date.
+   @composer update --no-progress --lock
```